### ISSUE TITLE
Missing grass colors (just a copy of sea)

### ISSUE THREAD

**Issue by kloon15** [eluleci/FlatUI#12]
As the title suggests, there are no grass colors, just a copy of the sea colors.

```
<!-- grass -->
<color name="grass_darker">#1a3b6c</color>
<color name="grass_dark">#1c52a2</color>
<color name="grass_primary">#2d72d9</color>
<color name="grass_light">#d5e3f7</color>

<!-- sea -->
<color name="sea_darker">#1a3b6c</color>
<color name="sea_dark">#1c52a2</color>
<color name="sea_primary">#2d72d9</color>
<color name="sea_light">#d5e3f7</color>
```

**Comment by eluleci** [comment]
I must have forgotten it during the refactoring. I'll fix it soon. Sorry

**Comment by eluleci** [comment]
I fixed this in version 2.0.1. Thanks for letting me know
